highfidelity/hifi_tests
# Add protocol tests

# Description
Adds a number of protocol sanity tests: Polvox, Text and Sphere
# Testing
1.  Start interface from the command line (build number is not important): `interface.exe --url "hifi://localhost" --no-login-suggestion --no-updater --testScript https://raw.githubusercontent.com/NissimHadar/hifi_tests/addPolyvoxProtocolTest/tests/protocol/testRecursive.js quitWhenFinished --testResultsLocation xxxx`, where xxxx is an empty folder.
3.  Verify that all images ending in `00001.png` show a green box.
5.  Verify that each of the  relevant  text files ends  in `.passed.txt`.
